src/font/font_face.rs

```diff
@@ -26,16 +26,14 @@ pub struct FontFace(ptr::NonNull<ffi::cairo_font_face_t>);
 
 impl FontFace {
     pub fn toy_create(family: &str, slant: FontSlant, weight: FontWeight) -> FontFace {
```

Comment:
What will happen here if you pass an invalid font family?

Reply:
Hmm I somehow completely forgot to ask myself that ^^